src/main/java/seedu/address/logic/parser/ArgumentTokenizer.java

```diff
@@ -87,7 +88,7 @@ private static int findPrefixPosition(String argsString, String prefix, int from
     private static ArgumentMultimap extractArguments(String argsString, List<PrefixPosition> prefixPositions) {
 
         // Sort by start position
-        prefixPositions.sort((prefix1, prefix2) -> prefix1.getStartPosition() - prefix2.getStartPosition());
+        prefixPositions.sort(Comparator.comparingInt(PrefixPosition::getStartPosition));
```

Comment:
@macareonie Some of these changes are not strictly needed for the Java17 migration. If you think they are useful to have nevertheless, you can send separate PRs for them.